templates/powershell.txt

```diff
@@ -27,6 +27,12 @@ function global:__zoxide_pwd {
 
 # cd + custom logic based on the value of _ZO_ECHO.
 function global:__zoxide_cd($dir, $literal) {
+
+    if ($dir.StartsWith('%') -and $dir.EndsWith('%')) {
+        $envVar = $dir.Trim('%')
+        $dir = $env:$envVar
```

Comment:
I've tested on this, but your code seems to be generating errors when I load the profile on my machine.

I think you meant to write:
```
        $dir = ${env:$envVar}
```

I tried to fix this too and comes up with a different approach (at #911), which might be covering more cases than your solution. Looking for reviews too.

Reply:
Upon reviewing the repository locally on my machine, I realized the fix could be related to the powershell.txt file, specifically within the __zoxide_cd function, but since I'm currently working on a Linux machine, I wasn't able to test it. I apologize for submitting the PR without proper testing.